perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh

```diff
@@ -39,17 +39,38 @@ then
   UPLOAD_FLAGS="--upload_gs"
 fi
 
+run_load_test_and_fetch_metrics(){
+  # Executing perf tests
+  ./run_load_test_and_fetch_metrics.sh "$1" "$2" "$3" "$4" "$5"
```

Comment:
Add comment what "$1" "$2" "$3" "$4" "$5" refer

Reply:
Better to add const 
e.g.
X=$1

Reply:
Added.